vendor/brave-ios/Ads/BATAdsConfirmationType.h

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSInteger, BATAdsConfirmationType) {
+  BATAdsConfirmationTypeUndefined,  // = ads::ConfirmationType::kUndefined
+  BATAdsConfirmationTypeServed,     // = ads::ConfirmationType::kServed
+  BATAdsConfirmationTypeClicked,    // = ads::ConfirmationType::kClicked
+  BATAdsConfirmationTypeDismissed,  // = ads::ConfirmationType::kDismissed
+  BATAdsConfirmationTypeViewed,     // = ads::ConfirmationType::kViewed
+  BATAdsConfirmationTypeLanded,     // = ads::ConfirmationType::kLanded
+  BATAdsConfirmationTypeFlagged,    // = ads::ConfirmationType::kFlagged
+  BATAdsConfirmationTypeUpvoted,    // = ads::ConfirmationType::kUpvoted
+  BATAdsConfirmationTypeDownvoted,  // = ads::ConfirmationType::kDownvoted
+  BATAdsConfirmationTypeConversion  // = ads::ConfirmationType::kConversion
+} NS_SWIFT_NAME(ConfirmationType);
```

Comment:
This was incorrect prior, can we make it `AdsConfirmationType`

Reply:
Fixed